SpinCore_pp/field_control.py

```diff
@@ -0,0 +1,24 @@
+from Instruments.XEPR_eth import xepr
+
+
+def set_field(argv, config_dict):
+    """Ensures the field is appropriate before setting the desired field."""
+    if len(argv) == 2 and argv[1] == "stayput":
```

Comment:
I think do this outside the function.  It's weird to pass Argo, since it's so specific to the script.  So move this to the script, and call the function inside the if block.